### DISCUSSION TITLE
I would like use ENTER as SAVE QSO

### DISCUSSION THREAD

**Discussion by michalsiman** [foldynl/QLog#234] [Ideas]
I would like use keyboard for SAVE current working QSO - it is possible? Sometimes using mouse isnt to nice, please, allow ENTER key for SAVE current QSO ... thx

**Comment by foldynl** [comment]
**Short answer**: save your mouse buttons, use CTRL+s

**Long answer**: it is not possible to use ENTER with the current configuration, because ENTER already has its function at the level of the Callsign field (it sends a request to the Callbook). It would be possible to add ENTER on other fields, but in my opinion it would be confusing for the user that ENTER on one field does something different than ENTER on another field. That's why I'd rather stay with CTRL+s. But if you see other possibilities, every idea is welcome and can be included to QLog.

**Comment by michalsiman** [comment]
So sorry, CTRL+S is perfect. Thank you. Maybe will be nice, if the CTRL+S
possibility will be shown when mouse stay on SAVE button .... but it is
only detail.

Thank you so much for your logger, looks very good and i like it from first
look.

Michal Siman
(OK1SIM)



st 2. 8. 2023 v 11:26 odesílatel Ladislav ***@***.***> napsal:

**Comment by foldynl** [comment]
No problem. For more details, see https://github.com/foldynl/QLog/wiki/Shortcuts . There is a table of all defined shortcuts.Maybe it will help you.
